docs/features/rgb_matrix.md

```diff
@@ -589,13 +589,13 @@ This function can only be run from within an effect or indicator callback, other
 
 #### Arguments {#api-rgb-matrix-set-color-arguments}
 
- - `uint8_t index`  
+ - `uint8_t index`
```

Comment:
The trailing spaces here are intentional, please don't remove them.

Reply:
I can replace them with `<br />` to get a linebreak and to avoid removing them later.